src/AspNetCoreModuleV2/CommonLib/CommonLib.vcxproj

```diff
@@ -100,6 +100,7 @@
       <AdditionalIncludeDirectories>..\iislib;</AdditionalIncludeDirectories>
       <MultiProcessorCompilation>true</MultiProcessorCompilation>
       <LanguageStandard>stdcpp17</LanguageStandard>
+      <ForcedIncludeFiles>stdafx.h</ForcedIncludeFiles>

```

Comment:
Why are you force including these?

Reply:
So I don't have to include in every cpp file.